pyerrors/fits.py

```diff
@@ -112,6 +141,8 @@ def func(a, x):
         The possible methods are the ones which can be used for scipy.optimize.minimize and
         migrad of iminuit. If no method is specified, Levenberg-Marquard is used.
         Reliable alternatives are migrad, Powell and Nelder-Mead.
+    tol: float, optional
+        can only be used for combined fits and methods other than Levenberg-Marquard
```

Comment:
For future reference: We should explain the impact of this parameter and it should be considered in all fits where the tolerance of the fit method may be adjusted by the user.